charts/nexus-iq/values.yaml

```diff
@@ -19,6 +19,9 @@ iq:
   # add this line with this file path and the `licenseSecret` above to autoconfigure licensing
   # licenseFile: /etc/nexus-iq-license/license_lic
   extraLabels:
+  env:
+    - name: JAVA_OPTS
+      value: "-Djava.util.prefs.userRoot=${SONATYPE_WORK}/javaprefs"
```

Comment:
If someone sets their own JAVA_OPTS enc var, they also have to include the defaults listed here? Is there a way to both set the entire value, or alternately prepend to the existing default?

Reply:
not that I'm aware of. This is the default and it can be overridden so I think we're good. What I put in here is what is in the container already